install-appliance.md

```diff
@@ -116,9 +156,13 @@ The appliance requires a few additional tools.
 
 After the installation, access `https://your-machine-address`. This web page provides status information for the following initialisation process.
 
+> **NOTE:** By default, the Anbox Cloud Appliance uses self-signed certificates, which might cause a security warning in your browser. Use the mechanism provided by your browser to proceed to the web page.
```

Comment:
@aguadoenzo should we say something on how to do this on Chrome on Mac?

Reply:
I suggest to add a more detailed explanation to https://anbox-cloud.io/docs/manage/web-dashboard (since this doesn't apply only to the appliance) and link to that section from here (once we have the section).

Reply:
Agreed with @ru-fu. Alternatively we could link to a 3rd party blog post explaining this but I'm not sure this is the best idea

Reply:
We have a task to track this, so let's keep that update separate from this PR.